include/pagmo/algorithms/cmaes.hpp

```diff
@@ -387,7 +387,8 @@ class cmaes
             // 6 - Update evolution paths
             ps = (1. - cs) * ps + std::sqrt(cs * (2. - cs) * mueff) * invsqrtC * (mean - meanold) / sigma;
             double hsig = 0.;
-            hsig = (ps.squaredNorm() / N / (1. - std::pow((1. - cs), (2. * counteval / static_cast<double>(lam)))))
+            hsig = (ps.squaredNorm() / N
+                    / (1. - std::pow((1. - cs), (2. * static_cast<double>(counteval) / static_cast<double>(lam)))))
```

Comment:
it's done by the operator, but the implicitly-generated conversion triggers a warning on GCC in debug mode.